### PR TITLE
Subclass sync: remove `pip install oaklib`

### PR DESCRIPTION
**Pull request by joeflack4**
## Overview
Removes `oaklib` `pip install` from subclass-sync goal, as ODK is now up-to-date.

### Documentation
Was the documentation added/updated under `docs/`?
- [ ] Yes
- [x] No, updates to the docs were not necessary after careful consideration

### QC
Was the full pipeline run before submitting this PR using `sh run.sh make build-mondo-ingest` on this branch (after 
`docker pull obolibrary/odkfull:dev`), and no errors occurred?
- [ ] Yes
- [ ] No, there are no functional (code-related) changes to the pipeline in the PR, so no re-run is necessary
   
### New Packages
Were any new *Python packages* added?
- [ ] Yes, [requirements.txt.full](https://github.com/INCATools/ontology-development-kit/blob/master/requirements.txt.full) was updated and an [ODK issue](https://github.com/INCATools/ontology-development-kit/issues/new) submitted
- [x] No

Were any other *non-Python packages* added?
- [ ] Yes, [Dockerfile](https://github.com/INCATools/ontology-development-kit/blob/master/Dockerfile) updated and an [ODK issue](https://github.com/INCATools/ontology-development-kit/issues/new) submitted
- [x] No

### PR Review and Conversations Resolved
Has the PR been sufficiently reviewed by at least 1 team member of the Mondo Technical team and all threads resolved?
- [ ] Yes

### CODE REVIEW

File: src/ontology/mondo-ingest.Makefile
Comment (joeflack4) [inline src/ontology/mondo-ingest.Makefile:529]: As the `todo` comment above says, this was only necessary so long as ODK's version of `oaklib` was < 0.5.20. This is now true for both the `dev` and `latest` containers.

```
docker run obolibrary/odkfull:latest pip freeze | grep oaklib
oaklib==0.5.25

docker run obolibrary/odkfull:dev pip freeze | grep oaklib
oaklib==0.6.5
sh run.sh pip freeze | grep oaklib
oaklib==0.6.5
```

I think this is one of the examples where this doesn't require a build before merging into `develop`. It can be double checked at `develop`->`main` pre-merge build time. But I will leave it up to you guys.

**Review by matentzn — APPROVED** [review]

**Pull request merged**
